types/rsvp/index.d.ts

```diff
@@ -54,192 +54,224 @@ declare module 'rsvp' {
-                T9 | PromiseLike<T9>,
-                T10 | PromiseLike<T10>
-            ]
-        ): RSVPPromise<[T1, T2, T3, T4, T5, T6, T7, T8, T9, T10]>;
-        function all<T1, T2, T3, T4, T5, T6, T7, T8, T9>(
-            values: [
-                T1 | PromiseLike<T1>,
-                T2 | PromiseLike<T2>,
-                T3 | PromiseLike<T3>,
-                T4 | PromiseLike<T4>,
-                T5 | PromiseLike<T5>,
-                T6 | PromiseLike<T6>,
-                T7 | PromiseLike<T7>,
-                T8 | PromiseLike<T8>,
-                T9 | PromiseLike<T9>
-            ]
-        ): RSVPPromise<[T1, T2, T3, T4, T5, T6, T7, T8, T9]>;
-        function all<T1, T2, T3, T4, T5, T6, T7, T8>(
-            values: [
-                T1 | PromiseLike<T1>,
-                T2 | PromiseLike<T2>,
-                T3 | PromiseLike<T3>,
-                T4 | PromiseLike<T4>,
-                T5 | PromiseLike<T5>,
-                T6 | PromiseLike<T6>,
-                T7 | PromiseLike<T7>,
-                T8 | PromiseLike<T8>
-            ]
-        ): RSVPPromise<[T1, T2, T3, T4, T5, T6, T7, T8]>;
-        function all<T1, T2, T3, T4, T5, T6, T7>(
-            values: [
-                T1 | PromiseLike<T1>,
-                T2 | PromiseLike<T2>,
-                T3 | PromiseLike<T3>,
-                T4 | PromiseLike<T4>,
-                T5 | PromiseLike<T5>,
-                T6 | PromiseLike<T6>,
-                T7 | PromiseLike<T7>
-            ]
-        ): RSVPPromise<[T1, T2, T3, T4, T5, T6, T7]>;
-        function all<T1, T2, T3, T4, T5, T6>(
-            values: [
-                T1 | PromiseLike<T1>,
-                T2 | PromiseLike<T2>,
-                T3 | PromiseLike<T3>,
-                T4 | PromiseLike<T4>,
-                T5 | PromiseLike<T5>,
-                T6 | PromiseLike<T6>
-            ]
-        ): RSVPPromise<[T1, T2, T3, T4, T5, T6]>;
-        function all<T1, T2, T3, T4, T5>(
-            values: [
-                T1 | PromiseLike<T1>,
-                T2 | PromiseLike<T2>,
-                T3 | PromiseLike<T3>,
-                T4 | PromiseLike<T4>,
-                T5 | PromiseLike<T5>
-            ]
-        ): RSVPPromise<[T1, T2, T3, T4, T5]>;
-        function all<T1, T2, T3, T4>(
-            values: [
-                T1 | PromiseLike<T1>,
-                T2 | PromiseLike<T2>,
-                T3 | PromiseLike<T3>,
-                T4 | PromiseLike<T4>
-            ]
-        ): RSVPPromise<[T1, T2, T3, T4]>;
-        function all<T1, T2, T3>(
-            values: [
-                T1 | PromiseLike<T1>,
-                T2 | PromiseLike<T2>,
-                T3 | PromiseLike<T3>
-            ]
-        ): RSVPPromise<[T1, T2, T3]>;
-        function all<T1, T2>(
-            values: [T1 | PromiseLike<T1>, T2 | PromiseLike<T2>]
-        ): Promise<[T1, T2]>;
-        function all<T>(values: (T | PromiseLike<T>)[]): RSVPPromise<T[]>;
+        const all: typeof Promise.all;
```

Comment:
This is a good catch – I wasn't sure if they were all on the promise prototype or not. 👍